components/build/tekton-chains/chains-controller-deployment.yaml

```
@@ -8,9 +8,8 @@ spec:
     spec:
       containers:
       - name: tekton-chains-controller
-        # Contract team fork may be useful
-        # in future for testing features not yet upstream:
-        # image: ghcr.io/hacbs-contract/chains/controller@sha256...
+        # Use a custom version of chains that contains support for pipelinerun attestations
+        image: ghcr.io/hacbs-contract/chains/controller:abf66a88e1d0497bcd751de7c3dd0e077766a56e@sha256:ccbf0d5ceb99a3aa074466423e77bf5f72976ac80abf94e15e91cc0fca13eae7
```

I think using `poc-tep-84` tag would make the intent clearer:

```suggestion
        image: ghcr.io/hacbs-contract/chains/controller:poc-tep-84@sha256:ccbf0d5ceb99a3aa074466423e77bf5f72976ac80abf94e15e91cc0fca13eae7
```